catalog/nsili/catalog-nsili-bqs/src/main/antlr4/org/codice/alliance/nsili/common/grammar/Bqs.g4

```diff
@@ -1,5 +1,64 @@
 grammar Bqs;
+/*
+STANAG 4559 BQS GRAMMAR
 
+query ::= term { "or" term }
+term ::= factor { "and" factor }
+factor ::= ["not"] primary
+primary ::= ( simple_attribute_name comp_op constant_expression )
+( geo_attribute_name geo_op geo_element)
+| ( geo_attribute_name rel_geo_op number dist_units “of” geo_element) | ( text_attribute_name "like" quoted_string )
+| ( attribute_name “exists”) | ( "(" query ")" )
+attribute ::= a member of the set of queryable attribute names (defined in the appropriate GIAS profile)
+attribute_name ::= attribute | {entity “:”}entity “.” attribute
+simple_attribute_name ::= member of subset of attribute_name for which boolean operators (comp_op) are allowed
+geo_attribute_name ::= member of subset of attribute_name for which geospatial operators are allowed
+text_attribute_name ::= member of subset of attribute_name for which string operators are allowed (“free text search”)
+comp_op ::= "=" | "<" | ">" | "<>" | "<=" | ">="
+constant_expression ::= number | quoted_string | date
+date ::= “’” year “/” month “/” day [“<blank>” hour “:” minute “:” second]”’” year ::= digit digit digit digit
+month ::= digit digit
+day ::= digit digit
+hour ::= digit digit minute ::= digit digit
+  F-2
+Amendment 2
+second ::= digit digit [“.” digit {digit}]
+geo_op ::= “intersect” | “outside”| “inside”
+rel_geo_op ::= “within” | “beyond”
+dist_units ::= “feet” | “meters” | “statute miles” | “nautical miles” | “kilometers”
+geo_element ::= point | polygon | rectangle | circle | ellipse | line | polygon_set
+sign ::= “+” | “-”
+number ::= [sign] digit_seq [ "." [ digit_seq ] ]
+digit_seq ::= digit { digit }
+digit ::= “0” | “1” | “2” | “3” | “4” | “5” | “6” | “7” | “8” | ” 9”
+quoted_string ::= "'" { character } "'" // Single quotes
+character ::= “a”|”b”| .... // All printable ASCII characters To use a "'" (single quote) use "''" (two single quotes)
+Del = “,” // Delimiter
+latitude ::= number
+longitude ::= number
+hemi ::= “N” | “S” | “E” | “W”
+DMS ::= [digit] digit digit “:” digit digit “:” digit digit “.” digit hemi latitude_DMS ::= DMS
+longitude_DMS ::= DMS
+latlon ::= latitude Del longitude | latitude_DMS Del longitude_DMS
+coordinate ::= latlon
+point :: = “POINT” “(“ coordinate “)”
+polygon ::= “POLYGON” “(“ coordinate Del coordinate Del coordinate {Del
+coordinate}“)”
+rectangle ::= “RECTANGLE” “(“ upper_left Del lower_right “)”
+upper_left ::= coordinate
+lower_right ::= coordinate
+circle ::= “CIRCLE” “(“ coordinate Del radius “)”
+radius ::= number dist_units
+ellipse ::= “ELLIPSE” “(“ coordinate Del major_axis_len Del minor_axis_len Del north_angle “)”
+major_axis_len ::= number dist_units minor_axis_len ::= number dist_units north_angle ::= number
+number is given in decimal degrees north is 0 and direction is clockwise range is 0 to 360
+line ::= “LINE” “(“ coordinate Del coordinate { Del coordinate} “)” polygon_set ::= “POLYGON_SET” “(“ polygon { Del polygon} “)”
+*/
+
+options {
+   k=10;
+   debugOutput=true;
```

Comment:
I added this to see if it made a difference, but didn't notice it when running. Have you seen it increase the debug output? If not, we might just want to remove it.